### DISCUSSION TITLE
Issue with conda activate

### DISCUSSION THREAD

**Discussion by Teptnologic** [pkgxdev/pkgx#900] [Help]
Hello, not sure if this is a viable use case. I am trying to run the `conda` package using pkgx.

```
pkgx conda create -n myenv -y python=3.10.6
pkgx conda activate myenv
```

The first command runs fine and I can run `pkgx conda --help` and `pkgx conda init`. 

However, I cannot get `pkgx conda activate` to work.

The error that I am getting: 
```
usage: conda [-h] [-v] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'activate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'doctor', 'env')
```

**Comment by jhheider** [comment]
their docs say:
```
Note

conda activate only works on conda 4.6 and later versions.

For conda versions prior to 4.6, type:

Windows: activate snowflakes

macOS and Linux: source activate snowflakes

Now that you are in your snowflakes environment, any conda commands you type will go to that environment until you deactivate it.
```

all of our versions are well newer than that (23.x.y; they switched to semver in 2022), so that definitely _should_ work if their docs are up-to-date. have you tried `source activate myenv`?

**Reply by Aditya-doid** [reply]
Good replai

**Reply by Teptnologic** [reply]
yes, it didn't work. The conda version install via pkgx at the time was 23.9.0, so that was expected.

```
$ pkgx conda --version
conda 23.9.0     
```

**Comment by DamezNZ** [comment]
has this been solved yet?
